internal/app/wrapperd/main.go

```diff
@@ -78,26 +83,45 @@ func Main() {
 	} else if droppedCaps != "" {
 		caps := strings.Split(droppedCaps, ",")
 		dropCaps := slices.Map(caps, func(c string) cap.Value {
-			capability, err := cap.FromName(c)
-			if err != nil {
-				log.Fatalf("failed to parse capability: %v", err)
+			capability, capErr := cap.FromName(c)
+			if capErr != nil {
+				log.Fatalf("failed to parse capability: %v", capErr)
 			}
 
 			return capability
 		})
 
 		// drop capabilities
 		iab := cap.IABGetProc()
-		if err := iab.SetVector(cap.Bound, true, dropCaps...); err != nil {
+		if err = iab.SetVector(cap.Bound, true, dropCaps...); err != nil {
 			log.Fatalf("failed to set capabilities: %v", err)
 		}
 
-		if err := iab.SetProc(); err != nil {
+		if err = iab.SetProc(); err != nil {
 			log.Fatalf("failed to apply capabilities: %v", err)
 		}
 	}
 
+	if flagPassed(flagUID) {
```

Comment:
we could just use a normal flag and by default it would have a value of zero, so if not zero set it, makes it simple

Reply:
done